### ISSUE TITLE
Tracking issue - NVMe sleep glitch

### ISSUE THREAD

**Issue by cybik** [cybik/tk-quanta-baseline#6]
This is the dmesg report issue:

```
[   95.384499] nvme nvme0: I/O 20 QID 0 timeout, reset controller
[   95.385200] PM: dpm_run_callback(): pci_pm_resume+0x0/0xf0 returns -16
[   95.385214] nvme 0000:04:00.0: PM: failed to resume async: error -16
```

This is a wake glitch coming from the Zen3/Cézanne platform requiring a specific sleep state to be triggered. It already works on Windows 10 (because of course it does), but the fix is yet to be merged in the Linux kernel.

The DSDT patch found at https://github.com/foundObjects/GA503QR-StorageD3Enable-DSDT-Patch (concerning ASUS computers) is actually generic, but there is no need for the patch, as that specific _DSD acpi section is found in the Prometheus XVI's DSDT dump.

TLDR: Aw Yeh Quanta got this done.

**Comment by cybik** [comment]
The Kernel fix is queued for merge in 5.16, per this changelog: https://lkml.org/lkml/2021/11/21/282

Look for `libahci: Adjust behavior when StorageD3Enable _DSD is set`.

Issue will be closed when this gets tested by yours truly.
